extensions/amp-bind/0.1/test/test-bind-integration.js

```diff
@@ -162,6 +162,73 @@ describe.configure().retryOnSaucelabs().run('integration amp-bind', function() {
     });
   });
 
+  describe('amp-live-list integration', () => {
+    function createFromServer(childAttrs = []) {
```

Comment:
I see this is copied from `test-amp-live-list.js`. Generally not a good idea to dupe code, and besides many parts of it is unused here (`updateTime`, `tombstone`). 

You're only calling this method once so you can probably remove it and replace it with a single `innerHTML = ...` line.